HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -143,10 +142,10 @@ private void GoToPrevTip()
             }
 
             // Navigate to the previous tip.
-            bool success = NavigateToTip(previousTip, markAsSeen: false);
+            bool success = NavigateToTip(previousTip, markAsSeen: true);
```

Comment:
We don't want this change. When navigating to a previous tip, it we don't want to markAsSeen, as it will have already been seen.